.buildkite/build.rayci.yml

```diff
@@ -34,3 +34,13 @@ steps:
       - forge
       - raypy38cu118base
     job_env: forge
+
+  - label: ":tapioca: build: pip-compile dependencies"
+    instance_type: small
+    commands:
+      - rm ./python/requirements_compiled.txt
+      - ./ci/ci.sh compile_pip_dependencies
+      - cp -f ./python/requirements_compiled.txt /artifact-mount/var/
+    depends_on:
```

Comment:
mark it soft-fail so on the day the internet decides this won't compile, it won't block people merging their PRs

Reply:
Ok!

Reply:
Actually @can-anyscale should we keep it as hard fail to make sure incompatible dependencies are not merged?